### ISSUE TITLE
Update customize.md to work with v2.-1

### ISSUE THREAD

**Issue by sfranky** [saulpw/visidata#381]
Example globalCommand:
```
globalCommand('^D', 'scroll-halfpage-down', 'cursorDown(nVisibleRows//2); sheet.topRowIndex += nVisibleRows//2')
```
gives error:
```
Traceback (most recent call last):
  File "/home/sfranky/.local/lib/python3.7/site-packages/visidata/basesheet.py", line 114, in exec_command
    exec(code, vdglobals, LazyMap(vd, self))
  File "scroll-halfpage-down", line 1, in <module>
    import math
NameError: name 'nVisibleRows' is not defined
```

Running on version v2.-1

**Comment by anjakefala** [comment]
Hi @sfranky!

Nice catch, thanks. That is a v2.-1 doc bug, I am behind on the docs. =) @saulpw has recently upgraded how scrolling works: 4ba51a67f7. 

Those docs are for version v1.5.2. Since then, `nVisibleRows` was renamed to `nScreenRows`. I will keep this issue open for doc-upgrading, but those docs will not be shipped to the website until v2.0 is released. We are still exploring how the API will look for it!

**Comment by anjakefala** [comment]
This has been updated for the time being in https://github.com/saulpw/visidata/commit/cdccd9bae6fe4fd991e87a849997e2baee47c8ba.
